sig-usability/README.md

```diff
@@ -0,0 +1,52 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Usability Special Interest Group
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximally usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
```

Comment:
s/Thiis/this

Reply:
> Examples of efforts include internationalization and accessibility of documentation.

I'm glad to see internationalization get more visibility. Because projects use a variety of static site generators, and not just Hugo like k/website, it's a good idea to aggregate i18n information across multiple projects. 

I have a couple of concerns about where to keep information about accessibility: 

1. Info about docs accessibility needs to live in the [docs contributors guide](https://kubernetes.io/docs/contribute/start/), not in a separate silo. Maintaining dual sources of information usually leads to divergence and staleness.

2. I can, however, see a case for general accessibility guidelines, similar to general i18n guidelines. 

Whatever we do with accessibility, I want to make sure that we aren't maintaining dual sources of information. That's setting contributors up for failure.

Reply:
I added a note that how-to guides for accessible documentation were out of scope and covered by SIG-docs.